File: tests/julia/studentCode.jl

# Correct
function factorial(n)
    result = one(n)
    for i in 1:n
        result *= i
    end
    return result
end

# Wrong
#function factorial(n)
#    result = one(n)
#    for i in 1:n
#        result *= i + 1
#    end
#    return result
#end

File: tests/python/studentCode.py

# Correct
# def fib(n):
#    if n <= 1:
#        return n
#    else:
#        return(fib(n-1) + fib(n-2))

# Wrong
def fib(n):
   if n <= 2:
       return n
   else:
       return(fib(n-1) + fib(n-2))